specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2020-09-01-preview/inference.json

```
@@ -0,0 +1,1443 @@
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "MachineLearningInferenceEndpointDeployments"
+        ],
+        "description": "Creates or updates an Inference Deployment. This call will update an Inference Deployment if it exists. This is a nonrecoverable operation. If your intent is to create a new Inference Deployment, do a GET first to verify that it does not exist yet.",
+        "operationId": "MachineLearningInferenceEndpointDeployment_CreateOrUpdate",
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "Create Or Update Deployment": {
+            "$ref": "./examples/createOrUpdateDeployment.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/WorkspaceNameParameter"
+          },
+          {
+            "$ref": "#/parameters/APIVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/InferenceEndpointNameParameter"
+          },
+          {
+            "$ref": "#/parameters/InferenceEndpointDeploymentNameParameter"
+          },
+          {
+            "in": "body",
+            "name": "properties",
+            "description": "The payload that is used to create or update the Inference Deployment.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/InferenceEndpointDeploymentCreateRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Inference Deployment creation or update initiated.",
+            "schema": {
+              "$ref": "#/definitions/InferenceEndpointDeploymentResource"
+            }
+          },
+          "202": {
+            "description": "Inference Deployment creation or update initiated.",
+            "headers": {
+              "Azure-AsyncOperation": {
+                "description": "URI to poll for asynchronous operation status.",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/MachineLearningServiceError"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "MachineLearningInferenceEndpointDeployments"
+        ],
+        "description": "Updates mutable objects of an Inference Deployment. This call will update an Inference Deployment if it exists. This is a nonrecoverable operation. If your intent is to create a new Inference Deployment, call PUT or POST for creation.",
+        "operationId": "MachineLearningInferenceEndpointDeployment_Update",
```

Align with the List operationId
```suggestion
        "operationId": "MachineLearningInferenceEndpointDeployments_Update",
```